shared/src/business/useCases/docketEntry/addPaperFilingInteractor.ts

```diff
@@ -60,9 +59,7 @@ export const addPaperFilingInteractor = async (
 
   const isCaseConsolidationFeatureOn = await applicationContext
     .getUseCases()
-    .getFeatureFlagValueInteractor(applicationContext, {
-      featureFlag: ALLOWLIST_FEATURE_FLAGS.MULTI_DOCKETABLE_PAPER_FILINGS.key,
-    });
+    .getAllFeatureFlagsInteractor(applicationContext);
```

Comment:
It looks like `getAllFeatureFlagsInteractor` returns a `key: value` `object` instead of a boolean. Perhaps we could pass in an optional `featureFlag` to the interactor or make use of a useCaseHelper... not sure. It doesn't look compatible at first glance.

Reply:
I think this one was an oversight that we forgot to update! I noticed it right as I saw your comment (while updating the test files) - fixing it now